scripts/ff_ci_pr_build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Sure. Though the actual script is being proposed in PR ( https://github.com/conda-forge/conda-forge-build-setup-feedstock/pull/52 ). Maybe the docs should live there? This is just a wrapper so the CircleCI log doesn't look [gross]( https://github.com/conda-forge/staged-recipes/pull/2264#discussion_r97234198 ). Though I could see an argument for spelling this out for the sake of the log.

Originally spelled out as "fast finish", but shortened it as it felt to verbose. Then thought FF could be interpreted as "fast forward", which seems equally appropriate IMHO.